pkg/options/HTTPServer.go

```diff
@@ -1,5 +1,15 @@
 package options
 
+func (o *HttpServerOpt) Defaults() {
+	if Environment().IsDevelopment() {
+		// enable web console and remove username, password defaults
+		// if this is explicitly via ENV, it will override these defaults
+		o.WebConsoleEnabled = true
+		o.WebConsoleUsername = ""
+		o.WebConsolePassword = ""
+	}
```

Comment:
As explained in the docs:
> Console is enabled and open to everyone by default in development environment (`ENVIRONMENT=dev`) or can be explicitly enabled by setting `HTTP_SERVER_WEB_CONSOLE_ENABLED` environmental variable to `true`. Default values for username is "admin" and random characters for password to avoid potential security issues.

So -- this is DISABLED by default and is only enabled, and open to everyone if you are running public server in dev mode. Not much we can help you there.

Reply:
I suppose I should have read the docs :see_no_evil: